.gitignore

```diff
@@ -82,6 +82,15 @@ _UpgradeReport_Files/
 /deps/icu-tmp
 ./node_modules
 /android-toolchain/
+/deps/cjs-module-lexer/bench
+/deps/cjs-module-lexer/test
+/deps/cjs-module-lexer/include-wasm
+/deps/cjs-module-lexer/lib
+/deps/cjs-module-lexer/Makefile
+/deps/cjs-module-lexer/README.md
+/deps/cjs-module-lexer/build.js
+/deps/cjs-module-lexer/src/lexer.js
+/deps/cjs-module-lexer/.*
```

Comment:
nit: I know our `.gitignore` unfortunately has other dependency ignores, I'd prefer these to go into a `deps/cjs-module-lexer/.gitignore` file instead. This is not blocking, however

Reply:
Will do.